pkg/transforms/filter.go

```diff
@@ -192,7 +201,12 @@ func (f *Filter) doEventFilter(filterProperty string, value string, lc logger.Lo
 	}
 
 	for _, name := range f.FilterValues {
-		if value == name {
+		item, err := regexp.Compile(name)
+		if err != nil {
+			lc.Errorf("bad regexp in filtering in pipeline '%s' : %s", f.ctx.PipelineId(), err.Error())
```

Comment:
Add the regexp, i.e. value of name.
```suggestion
			lc.Errorf("bad regexp (%s) in filtering in pipeline '%s' : %s", name, f.ctx.PipelineId(), err.Error())
```